pkg/logs/tekton_logging.go

```diff
@@ -194,79 +195,98 @@ func getPipelineRunNamesForActivity(pa *v1.PipelineActivity, tektonClient tekton
 		return tektonPRs.Items[i].CreationTimestamp.Before(&tektonPRs.Items[j].CreationTimestamp)
 	})
 
-	var names []string
+	runs := make(map[string]*v1alpha12.PipelineRun)
```

Comment:
why is the tekton pipeline import 'v1alpha12'. At best I would expect 'v1alpha1'. That said, I think it might be better to import it as something like 'pipeline' or 'tekton'. That helps me understand the code and is does not change with version upgrades. 'v1alpha12' is in this context just a "useless" version string which does not really help me making sense of the code.